js/logging.js

```diff
@@ -53,8 +53,8 @@ loggingModule.factory(
                 return false;
             }
 
-            if (LOGGING_CONFIG.FORWARD_TO_NEWRELIC && $window.NREUM && $window.NREUM.noticeError) {
-                $window.NREUM.noticeError(exception, { url: $window.location.href });
+            if (LOGGING_CONFIG.FORWARD_TO_INSTANA && typeof ineum !== undefined && ineum) {
+                ineum('reportError', exception);
             }
```

Comment:
It's all a bit academic as we're the only ones who use this library, but I still wonder if these changes should be in addition to NewRelic support instead of as a replacement

Reply:
I think as it's just us and we hopefully wont be using it be the end of the year, I'm going to leave it as is :)